### PR TITLE
Refactor request-vc CLI

### PR DESCRIPTION
**Pull request by Kailai-Wang**
### Context

A small refactoring of request-vc CLI.

The reasons are:
- duplicate code in `request-vc` and `request-batch-vc` CLI (they were probably copied around)
- IMO the string comparison for `request-batch-vc` is too verbose and insufficient hint is given when entering the wrong parameter (type/count).
- https://github.com/litentry/litentry-parachain/pull/2595#pullrequestreview-1945716984

So this PR tries to:
- provide only one CLI (single vc can be done via batching with size 1)
- replace string comparison with subcommand
- improves the interface a bit, hoping it to be clearer

Basically it looks like this now: 
```
<binary> trusted -d request-vc <did> [-s] -a "<value>" -a "<value>" ...
```

Example of batched vc a1 + a2 + a3:
```
./bin/litentry-cli trusted -d request-vc \
  did:litentry:substrate:0x8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48 \
  -a "a1" -a "a2 gid" -a "a3 gid cid rid"
```
see the comment in the code for more examples

### CODE REVIEW

**Review by BillyWooo — APPROVED** [review]
Thanks. Very nice refactoring. 
Maybe we don't need option `stf: bool` anymore.

**Review by felixfaisal — APPROVED** [review]
LGTM
